base/compiler/abstractinterpretation.jl

```diff
@@ -3314,17 +3314,15 @@ function abstract_eval_binding_partition!(interp::AbstractInterpreter, g::Global
     return partition
 end
 
+const generic_getglobal_effects = Effects(EFFECTS_TOTAL; consistent=ALWAYS_FALSE, nothrow=false, inaccessiblememonly=ALWAYS_FALSE)
```

Comment:
`generic_getglobal_effects` is defined at L2293, so this definition is duplicated.
```suggestion
```